controller/controllerutil.go

```diff
@@ -0,0 +1,78 @@
+package controller
+
+/*
+Copyright 2017 - 2020 Crunchy Data Solutions, Inc.
```

Comment:
Please make it `Copyright 2020`

Reply:
Changed to `Copyright 2020 Crunchy Data Solutions, Inc.`